perftest/perftest.go

```diff
@@ -25,7 +27,7 @@ type Opts struct {
 
 var (
 	dataNode dnWrapper
-	wallet WalletWrapper
+	wallet   WalletWrapper
```

Comment:
`dataNode` and `wallet` should probably not be global, but better as fields of a struct